compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/platform/DesktopInputComponentTest.kt

```diff
@@ -41,7 +41,7 @@ class DesktopInputComponentTest {
         val processor = EditProcessor()
 
         val input = DesktopTextInputService(PlatformComponent.Empty)
-        val inputService = TextInputService(input)
+        val inputService = TextInputService(input.asPlatformTextInputService())
```

Comment:
If we don't take my other comment into account (about undeprecation `TextInputService`), and if we would go a designing a new API way, then we would need to completely remove the link between `ui` and `foundation` via the deprecated `TextInputService`.

In the current code all `foundation` implementation still uses the deprecated `TextInputService`, but it should use the new API `PlatformContextTextInputService`. This is the point of providing a stable new API. If we need it just for `ComposeScene`, then there is no need in `public`, we can use `@InternalComposeUiApi`.